docs/en/sql-reference/sql-statements/data-definition/CREATE_TABLE_AS_SELECT.md

```diff
@@ -45,11 +49,13 @@ You can submit an asynchronous CTAS task using [SUBMIT TASK](../data-manipulatio
 
 | **Parameter**     | **Required** | **Description**                                              |
 | ----------------- | ------------ | ------------------------------------------------------------ |
-| column_name       | Yes          | The name of a column in the new table. You do not need to specify the data type for the column. StarRocks automatically specifies an appropriate data type for the column . StarRocks converts FLOAT and DOUBLE data into DECIMAL(38,9) data. StarRocks also converts CHAR, VARCHAR, and STRING data into VARCHAR(65533) data. |
+| column_name       | No          | The name of a column in the new table. You do not need to specify the data type for the column. StarRocks automatically specifies an appropriate data type for the column . StarRocks converts FLOAT and DOUBLE data into DECIMAL(38,9) data. StarRocks also converts CHAR, VARCHAR, and STRING data into VARCHAR(65533) data. |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/86dadf1a-3848-45d6-8184-f43194241a25)
remove the sapce